### PR TITLE
New "About" category with SLA page

### PR DESCRIPTION
**Pull request by cullywakelin**
Created a new category "About" with a page dedicated to describing the documentation SLA.

### CODE REVIEW

File: docs/docs-sla.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+id: docs-sla
+title: Documentation SLA
+---
+
+This documentation set and all of its components, such as page urls, content, and design, is subject to change at any time.
```
Comment (rylandg) [inline docs/docs-sla.md:6]: I think this is very ambiguously worded. What is a "documentation set"? I would try rephrase this in a less "legal" way ie:

We are constantly working to improve and expand our documentation. There is no guarantee that links will remain active after the documentation is revised. ....

File: docs/docs-sla.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+Visit [community.temporal.io](https://community.temporal.io) for help if the information you are looking for appears to be missing.
+
+View the [source repository README](https://github.com/temporalio/documentation-legacy) for contribution guidelines if you wish to make changes.
```
Comment (rylandg) [inline docs/docs-sla.md:10]:
```suggestion
Feel like helping improve the Temporal docs experience? Visit [source repository README](https://github.com/temporalio/documentation-legacy) to see our contribution guidelines.
```

File: docs/docs-sla.md (hunk 1 above)
Comment (cullywakelin) [reply docs/docs-sla.md]: I revamped the tone and context of all this content - LMK what you think.

File: docs/docs-sla.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+id: docs-sla
+title: Documentation SLA
+---
+
+Temporal is constantly working to improve and expand its documentation. As a result, some components may change without notice. Page slugs (URLs), menu labels, and the location of information are a few of the items you can expect to see altered as we aim to try and give our users the best experience possible.
```
Comment (rylandg) [inline docs/docs-sla.md:6]: This is exactly what I wanted. Thank you

File: docs/docs-sla.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+[Documentation versions](https://docs.temporal.io/versions) are used to preserve changes to user facing content, which enables users to reference older material.
```
Comment (rylandg) [inline docs/docs-sla.md:8]: I think I would remove one use of the word "users"

**Pull request merged**